harvard-lts/etd_dash_service
# Etd 166 addons

**correct name for alma task; upgrade python.**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/ETD-166)

# What does this Pull Request do?
The task name specified for publishing to the alma service was incorrect, that is corrected here; also updated python to 3.11

# How should this be tested?

Start docker locally (see readme)
Exec into docker (see readme)
run > pytest test/unit

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N - no change in functionality
- integration tests? N
